### PR TITLE
Updated language for Empty HH Account health check error

### PR DESCRIPTION
**Pull request by sam-knox**
# Critical Changes

# Changes
We updated the language in the Empty Household Account health check message to point to the correct health check report. Previously, the error message pointed to the Household object report.

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by sam-knox** [comment]
@RadSFDeveloper - Would you mind doing a code review for this simple label change?

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -4103,7 +4103,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>healthSolutionHHAccNoContacts</shortDescription>
-        <value>Consider deleting these unused Household object records to save space. If you have completed a conversion to the Household Account model and no longer need the data on the Household object records, you can safely delete them. View these Household object records by running the Empty Household Objects report in the NPSP Health Check reports folder.</value>
+        <value>Consider deleting these unused Household Account records to save space. View these Household Account records by running the &lt;b&gt;Empty Household Accounts&lt;/b&gt; report in the NPSP Health Check reports folder.</value>
```
Comment (RadSFDeveloper) [inline src/labels/CustomLabels.labels]: @sam-knox , this change is verified fine (see screenshot below). There is another label with similar text, I am wondering if `healthSolutionHHObjNoContacts` requires any change?

Also, if this PR is referencing a WI, can you use `**lurch: attach W-....` to connect them?

Screen Shot 2019-07-17 at 8 47 24 PM

**Comment by RadSFDeveloper** [comment]
**lurch: attach W-033612

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E0000087vQ7QAI" target="_blank">W-033612</a>

**Pull request merged**